### DISCUSSION TITLE
How to run gauge specs from command line for a maven-java project created in Intellij.

### DISCUSSION THREAD

**Discussion by preetipadalia** [getgauge/gauge#1885] [General]
Hi,

I am creating gauge project in Intellij and added maven support. The execution is happening through intellij but unable to run/spec from command line. Can you please help me with this?

Thanks
Preeti

**Comment by zabil** [comment]
The documentation is available at https://github.com/getgauge-contrib/gauge-maven-plugin#executing-specs
